### DISCUSSION TITLE
Issues integrating it with firebase, please help

### DISCUSSION THREAD

**Discussion by sanchitgng** [lupidan/apple-signin-unity#126] [Q&A]
![image](https://user-images.githubusercontent.com/40668553/137709160-93d1e699-e37f-4f5e-a548-a51c535ab7b3.png)
Can anyone please explain how can I get the raw nonce using this plugin, I am integrating it with firebase, please help me. Thank you!

**Comment by lupidan** [comment]
Read the documentation please.
https://github.com/lupidan/apple-signin-unity#nonce-and-state-support-for-authorization-requests
